sauce/features/budget/toggle-master-categories/index.css

```diff
@@ -1,4 +1,4 @@
-.budget-toolbar > .undo-redo-container:after {
+.budget-toolbar > .undo-redo-container::after {
```

Comment:
Is the `::` intentional here? Suspect we meant `:after`.

Reply:
:after is CSS2 style, ::after is CSS3 style

Reply:
Guess I'm totally showing my age. In it goes, thanks!

Reply:
Um, yeah. Showing your age...lol